src/components/NewAnalysisModal.js

```diff
@@ -1,37 +1,112 @@
 import _ from 'lodash/fp'
 import { Fragment, useState } from 'react'
-import { div, h, hr, img } from 'react-hyperscript-helpers'
-import { spinnerOverlay } from 'src/components/common'
+import { div, h, h2, hr, img, span } from 'react-hyperscript-helpers'
+import { ButtonPrimary, IdContainer, Select, spinnerOverlay, WarningTitle } from 'src/components/common'
+import Dropzone from 'src/components/Dropzone'
+import { icon } from 'src/components/icons'
 import ModalDrawer from 'src/components/ModalDrawer'
 import { NewGalaxyModalBase } from 'src/components/NewGalaxyModal'
 import { NewRuntimeModalBase } from 'src/components/NewRuntimeModal'
-import { tools } from 'src/components/notebook-utils'
+import {
+  analysisNameInput,
+  analysisNameValidator,
+  getDisplayName, getTool,
+  notebookData,
+  tools
+} from 'src/components/notebook-utils'
```

Comment:
Could you put these imports on one line, as this takes up a lot of vertical space in the file